src/applications/HEDA.app

```diff
@@ -2,13 +2,16 @@
 <CustomApplication xmlns="http://soap.sforce.com/2006/04/metadata">
     <defaultLandingTab>standard-home</defaultLandingTab>
     <description>Higher Education Data Architecture</description>
+    <formFactors>Large</formFactors>
```

Comment:
@lifewithryan I am hesitating on these lines too. Not sure if we want to include them here..